main.js

```diff
@@ -1,84 +1,49 @@
+/* eslint-disable no-undef */
 /* eslint-disable class-methods-use-this */
 /* eslint-disable no-unused-vars */
-import { DateTime } from './modules/luxon.js';
-import Book from './modules/Book.js';
 
+// variables
 const addForm = document.querySelector('.form');
-const contactSec = document.querySelector('.contact');
 const showBooks = document.querySelector('.record');
 const Title = document.querySelector('.title');
 const Author = document.querySelector('.author');
 const btn = document.querySelector('.form button');
-const booksList = document.getElementById('booksList');
-const newBook = document.getElementById('newBook');
-const contactInfo = document.getElementById('contactInfo');
-const recordSec = document.querySelector('.record-section');
-const currentDate = DateTime.local();
-
-const newDate = currentDate.toLocaleString({
-  month: 'long',
-  day: 'numeric',
-  year: 'numeric',
-});
-const newTime = currentDate.toLocaleString(DateTime.TIME_WITH_SECONDS).toLowerCase();
-
-const bookc = new Book();
-
-document.getElementById('current-date').innerHTML = `${newDate}, ${newTime}`;
-
-booksList.addEventListener('click', () => {
-  window.location.reload();
-  recordSec.style.display = 'block';
-  addForm.style.display = 'none';
-  contactSec.style.display = 'none';
-});
-
-newBook.addEventListener('click', () => {
-  recordSec.style.display = 'none';
-  addForm.style.display = 'block';
-  contactSec.style.display = 'none';
-});
-
-contactInfo.addEventListener('click', () => {
-  recordSec.style.display = 'none';
-  addForm.style.display = 'none';
-  contactSec.style.display = 'block';
-});
 
 // create book object
 // add books
 btn.addEventListener('click', () => {
-  if (Title.value === '' || Author.value === '') {
-    document.querySelector('.form > span').textContent = 'All fiels are required!';
-  } else {
-    const bookObj = {
-      id: new Date().getUTCMilliseconds(),
-      title: Title.value,
-      author: Author.value,
-    };
-    bookc.addBook(bookObj);
-  }
+  const books = JSON.parse(localStorage.getItem('data')) == null ? [] : JSON.parse(localStorage.getItem('data'));
+  const bookObj = {
+    id: new Date().getUTCMilliseconds(),
+    title: Title.value,
+    author: Author.value,
+  };
+  books.push(bookObj);
+  localStorage.setItem('data', JSON.stringify(books));
+  window.location.reload();
 });
 
-// ***********
-
-window.remove = (id) => {
-  bookc.removeBook(id);
+// remove book
+const removeBook = (id) => {
+  const books = JSON.parse(localStorage.getItem('data')) === null ? [] : JSON.parse(localStorage.getItem('data'));
+  if (books.length > 0) {
+    const filteredBooks = books?.filter((i) => i?.id !== id);
+    localStorage.setItem('data', JSON.stringify(filteredBooks));
+    window.location.reload();
+  }
 };
```

Comment:
As part of the project requirements, you should use es6 import keywords but I haven't seen the usage of that and it is coming from the revert of the PR. Could you fix this to stay consistent with the milestone requirements?


Also, this is what your app currently looks like, I do not see the date displayed as expected. Kindly implement this to make the project more like the wireframe.

![errApp](https://user-images.githubusercontent.com/84930355/220898638-a86a99c5-2018-490a-af9e-c7bb29b92435.PNG)